### PR TITLE
Adds end-end test for indications

### PR DESCRIPTION
**Pull request by KSchopmeyer**
Adds a new end-end test for:

1. Managing indication subscriptions.
2. Receiving indications from the WBEM Server

This test is based on the OpenPegasus docker image defined for the other end-end tests.

See commit mesage for details

NOTE: This commit includes changes for the example pegasusindicaitonstest.py since basically the same code is in both the end to end test and the example and this made the two almost the same.

In order to get a consinstent pass of this test, I disabled python 2.7 and reduced python 3.7 to minimum.  We could have passed on the PACKAGE_LEVEL variable from Makefile but this establishes that the issues with quitting early are at least probably tied to python 2.7 and 3.7 We also know from earlier tests that it is probably tied to the minimum config.

I suggest we commit this and I have already filed a bug on the failures with python 2.7, etc.

### CODE REVIEW

File: docs/changes.rst [hunk 1]
```diff
@@ -36,6 +36,11 @@ Released: not yet
 
 **Enhancements:**
 
+* Add end-end test for indications where an OpenPegasus server in docker
```
Comment (andy-maier) [inline docs/changes.rst:39]: Change "docker" -> "Docker".

File: docs/changes.rst [hunk 2]
```diff
@@ -36,6 +36,11 @@ Released: not yet
...
+  container is the WBEM Server and pywbem defines the client that manages
```
Comment (andy-maier) [inline docs/changes.rst:40]: On "pywbem defines": That sounds strange to me. How about "pywbem is used as" or "pywbem is"?

File: docs/changes.rst [hunk 3]
```diff
@@ -36,6 +36,11 @@ Released: not yet
...
+  indication subsecriptions and receives the indications. This test uses the
```
Comment (andy-maier) [inline docs/changes.rst:41]: Typo: subsecriptions

File: tests/end2endtest/test_indications.py [hunk 4]
```diff
@@ -0,0 +1,568 @@
...
+    port = 5000
+    log = True
+    verbose = True
+
+    # define repeat_loop and indication_count variables to rerun the test with
+    # each of these pairs. (repeat_loop, indication_count)
+    # FUTURE: Expand this to more indications when we resolve issue #3022,
+    # the container not returning all of the indications.  This appears to
+    # be much more consistent with python 3.7 and 3.7 PACKAGE_LEVEL=minimum
+    if sys.version_info[1] == 7:   # Use minimum send ct while issue #3022
```
Comment (andy-maier) [inline tests/end2endtest/test_indications.py:70]: Can you please be specific about the major version? If you really mean this case to be taken for 2.7 and 3.7, then say:
```
if sys.version_info[0:2] in ((2, 7), (3, 7)):
```

Second: I do not understand how this version selection matches the comment about 3.7

**Review by andy-maier — CHANGES_REQUESTED** [review]

File: docs/changes.rst (hunk 2 above)
Comment (KSchopmeyer) [reply docs/changes.rst:40]: Changed to pywbem is both the client that creates indication subscriptions and the the indication listener

File: docs/changes.rst (hunk 3 above)
Comment (KSchopmeyer) [reply docs/changes.rst:41]: Fixed

File: tests/end2endtest/test_indications.py (hunk 4 above)
Comment (KSchopmeyer) [reply tests/end2endtest/test_indications.py]: Fixed.

**Review by andy-maier — APPROVED** [review]
